src/components/Homepage/HowItWorks.astro

```diff
@@ -1,4 +1,8 @@
-<section class="bg-neutral-900 px-4 sm:px-6 py-12 md:py-16 text-sm md:text-base">
+---
+import VerificationPhases from "../VerificationChain/VerificationPhases";
```

Comment:
I think `VerificationChain` and `VerificationPhases` should be something like `ChainVerificationAnimation`. Also, if you rename `VerificationPhases.tsx` to `index.tsx`, you should be able to import it as `import VerificationPhases from "../ChainVerificationAnimation";`